modules/grpc/otel/filterx/otel-logrecord.cpp

```diff
@@ -39,6 +40,63 @@
 
 using namespace syslogng::grpc::otel;
 
+/* C++ Implementations */
+class OtelConverter : public ProtobufField
+{
+public:
+  FilterXObject *FilterXObjectGetter(const google::protobuf::Message &message, ProtoReflectors reflectors)
+  {
+    std::string fieldName = reflectors.fieldDescriptor->name();
```

Comment:
This does an unnecessary copy.

Please use this instead:
```c++
const std::string &fieldName = reflectors.fieldDescriptor->name();
```

Same in the `Setter()`.